.github/copilot-instructions.md

```diff
@@ -0,0 +1,193 @@
+# GitHub Copilot Instructions for Blue Agent
+
+This file contains project-specific guidelines for GitHub Copilot to ensure code suggestions align with Blue Agent's mandatory conventions and best practices.
+
+## Security and Compliance
```

Comment:
Adds a section to specify the use, motivations and main features of the project. As well as the general constraints imposed.